chia/solver/solver.py

```diff
@@ -74,7 +75,14 @@ def get_connections(self, request_node_type: Optional[NodeType]) -> list[dict[st
         return default_get_connections(server=self.server, request_node_type=request_node_type)
 
     async def on_connect(self, connection: WSChiaConnection) -> None:
-        pass
+        if not self.config.get("trusted_peers_only", True):
+            self.log.info(f"trusted peers check disabled, Accepting connection from {connection.get_peer_logging()}")
+            return
```

Comment:
I would swap these check around, and make this log clearly state that we accepted a connection from an *untrusted* farmer